### ISSUE TITLE
"Understanding" - principle 1 definition

### ISSUE THREAD

**Issue by pfinch** [mozilla/olm-whitepaper#15]
The definition "understanding" appears narrow and the term seems superfluous.  Beyond "accessible", what does this convey?  

It appears to detract from some of the more important aspects of accessibility, such as IP licensing, unless this is covered under "Extensibility"?  It may be that I'm missing the general use case of the white paper, but for participatory projects, they're generally seeded with some asset released into a commons.  Is this considered covered under "gifting" within "Extensibility?"
